src/jennifer/adapter/base_sql_generator.cr

```diff
@@ -113,14 +113,14 @@ module Jennifer
 
       def self.union_clause(io, query)
         return unless query._unions
-        query._unions!.each do |u|
+        query._unions.not_nil!.each do |u|
```

Comment:
not really sure why the not_nil! is required here seeing as the ! is semantically the same but without it you get the no target defs issue.